src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="nav_link">
+        <img src="images/logo.png"
+          alt="MOYO-logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#samsung" class="nav_link">Samsung</a>
+          </li>
```

Comment:
Pay attention about BEM naming - it should have 2 underscore
`Block__element`
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a href="#apple" class="nav__link is-active">Apple</a>
          </li>
          <li class="nav__item">
            <a href="#samsung" class="nav__link">Samsung</a>
          </li>
```